.travis.yml

```diff
@@ -59,11 +60,12 @@ jobs:
   include:
 
     # This tests the formatting of a PR.
-    - if: (type = pull_request) OR (type = cron) OR NOT (branch = master)
+    - if: (type = pull_request) OR (type = cron) OR (branch = prerelease_test)
```

Comment:
This seems to have had the wrong logic since it triggers duplicate builds when one pushes a branch to the main repo and makes a pull request from there. (I.e., this used to build on both PR and updates to any branch except master).